sdk/healthinsights/azure-healthinsights-radiologyinsights/azure/healthinsights/radiologyinsights/_client.py

```diff
@@ -0,0 +1,100 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) Python Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+
+from copy import deepcopy
+from typing import Any
+
+from azure.core import PipelineClient
+from azure.core.credentials import AzureKeyCredential
+from azure.core.pipeline import policies
+from azure.core.rest import HttpRequest, HttpResponse
+
+from ._configuration import RadiologyInsightsClientConfiguration
+from ._operations import RadiologyInsightsClientOperationsMixin
+from ._serialization import Deserializer, Serializer
+
+
+class RadiologyInsightsClient(
+    RadiologyInsightsClientOperationsMixin
+):  # pylint: disable=client-accepts-api-version-keyword
+    """RadiologyInsightsClient.
+
+    :param endpoint: Supported Cognitive Services endpoints (protocol and hostname, for example:
+     https://westus2.api.cognitive.microsoft.com). Required.
+    :type endpoint: str
+    :param credential: Credential needed for the client to connect to Azure. Required.
+    :type credential: ~azure.core.credentials.AzureKeyCredential
+    :keyword api_version: The API version to use for this operation. Default value is
+     "2023-09-01-preview". Note that overriding this default value may result in unsupported
```

Comment:
FYI calling this out again since the api version in the readme doesnt match the generated one.

Reply:
changed yes